plugins/module_utils/network/sonic/config/login_lockout/login_lockout.py

```diff
@@ -0,0 +1,274 @@
+        warnings = list()
+        commands = list()
+
+        existing_login_lockout_facts = self.get_login_lockout_facts()
+
+        commands, requests = self.set_config(existing_login_lockout_facts)
+        if commands:
+            if not self._module.check_mode:
+                try:
+                    edit_config(self._module, to_request(self._module, requests))
+                except ConnectionError as exc:
+                    self._module.fail_json(msg=str(exc), code=exc.code)
+            result['changed'] = True
+        result['commands'] = commands
+
+        changed_login_lockout_facts = self.get_login_lockout_facts()
+
+        result['before'] = existing_login_lockout_facts
+
+        if result['changed']:
+            result['after'] = changed_login_lockout_facts
+        result['warnings'] = warnings
+
+        return result
+
+    def set_config(self, existing_login_lockout_facts):
+        """ Collect the configuration from the args passed to the module,
+            collect the current configuration (as a dict from facts)
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        want = self._module.params['config']
+
+        have = existing_login_lockout_facts
+
+        resp = self.set_state(want, have)
+
+        return to_list(resp)
+
+    def set_state(self, want, have):
+        """ Select the appropriate function based on the state provided
+
+        :param want: the desired configuration as a dictionary
+        :param have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        state = self._module.params['state']
+        diff = get_diff(want, have)
+        if state == 'deleted':
+            commands, requests = self._state_deleted(want, have, diff)
+        elif state == 'merged':
+            commands, requests = self._state_merged(diff)
+        elif state == 'replaced':
+            commands, requests = self._state_replaced_overridden(want, have, diff)
+        elif state == 'overridden':
+            commands, requests = self._state_replaced_overridden(want, have, diff)
+        return commands, requests
+
+    def _state_replaced_overridden(self, want, have, diff):
+        """ The command generator when state is replaced or overridden
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        requests = []
+        delete = {}
+        state = self._module.params['state']
+
+        for key in have:
+            if key in want:
+                if want[key] == have[key]:
+                    continue
+            if key not in diff:
+                delete[key] = have[key]
```

Comment:
Please handle cases when the 'default' value is configured and that option is not specified in want ( no deletion should be performed).